src/clr-angular/utils/drag-and-drop/droppable/droppable.ts

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {Directive, ElementRef, EventEmitter, Input, OnDestroy, OnInit, Output, Renderer2} from "@angular/core";
+import {Subscription} from "rxjs/Subscription";
+
+import {DomAdapter} from "../../dom-adapter/dom-adapter";
+
+import {ClrDragEvent, ClrDragEventType} from "../interfaces/drag-event";
+import {ClrDropTolerance} from "../interfaces/drop-tolerance";
+import {ClrDragAndDropEventBus} from "../providers/drag-and-drop-event-bus";
+
+@Directive({
+    selector: "[clrDroppable]",
+    providers: [DomAdapter],
+    host: {class: "droppable", "[class.draggable-match]": "isDraggableMatch"}
```

Comment:
You're clearing any class from the droppable element, use`[class.droppable]: 'true',`

Reply:
Actually he's not, I was really surprised by that too. `"class": "droppable"` does not clear, but `"[class]": "'droppable'"` does.

Still, I told him I thought this was a bit too obscure and had a risk of breaking in the future because it is undocumented.